recipes-core/images/core-image-pelux-minimal.bb

```diff
@@ -6,3 +6,4 @@
 DESCRIPTION = "Image for creating a Pelux image"
 
 inherit core-image-pelux
+IMAGE_INSTALL_remove_qemux86-64 = "template-service swupdate"
```

Comment:
It would be interesting to see why is the `template-service` a problem for the testimage builds? Is there any explanation. `swupdate` is quite big so I think it might affect the image but the `template-service` is nothing more than a `int main()` C++ executable.